interface/super/edit_list.php

```diff
@@ -396,7 +396,11 @@ function writeOptionLine($option_id, $title, $seq, $default, $value, $mapping=''
   } else {
     echo "  <td align='center' class='optcell'>";
     echo "<input type='text' name='opt[$opt_line_no][notes]' value='" .
-        htmlspecialchars($notes, ENT_QUOTES) . "' size='25' class='optin' />";
+      attr($notes) . "' size='25' maxlength='255' class='optin' ";
+    if ($list_id == 'lbfnames') {
+      echo "onclick='edit_layout_props($opt_line_no)' ";
+    }
```

Comment:
If you are redefining Notes field here (I think you are), makes sense to rename the Notes header to something more descriptive. Check out the code in this script for multiple examples of how this is done.